metrics/init.lua

```diff
@@ -126,7 +126,8 @@ return {
     enable_default_metrics = function(include, exclude)
         log.warn('metrics tnt_net_sent_rps, tnt_net_received_rps, tnt_net_connections_rps, '..
             'tnt_net_requests_rps, tnt_stats_op_rps, tnt_space_count, tnt_fiber_count, ' ..
-            'lj_gc_total, tnt_cpu_count, tnt_cpu_total, tnt_vinyl_scheduler_dump_count ' ..
+            'lj_gc_total, tnt_cpu_count, tnt_cpu_total, tnt_vinyl_scheduler_dump_count, ' ..
+            'tnt_replication_<id>_lag, tnt_replication_master/replica_<id>_lsn ' ..
```

Comment:
We must not introduce deprecated metrics

Reply:
What do you mean? Those metrics are deprecated, and new ones are added to replace them.

Reply:
Ouch
My bad, I incorrectly understood it.